app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -243,6 +245,14 @@ class PasswordFragment : Fragment(R.layout.password_recycler_view) {
         try {
             listener = object : OnFragmentInteractionListener {
                 override fun onFragmentInteraction(item: PasswordItem) {
+                    if (settings.getString(PreferenceKeys.SORT_ORDER) == PasswordRepository.PasswordSortOrder.RECENTLY_USED.name) {
+                        //save the time when password was used
+                        val preferences = context.getSharedPreferences("recent_password_history", Context.MODE_PRIVATE )
+                        val editor = preferences.edit()
+                        editor.putString(item.file.absolutePath, System.currentTimeMillis().toString())
+                        editor.apply()
```

Comment:
This will also need an import.
```suggestion
                        preferences.edit {
                            putString(item.file.absolutePath, System.currentTimeMillis().toString())
                        }
```